.github/workflows/torch.yml

```diff
@@ -99,7 +97,8 @@ jobs:
         BUILD_TORCH_VERSION=${{ inputs.torch-version }}
         BUILD_TORCH_VISION_VERSION=${{ inputs.torchvision-version }}
         BUILD_TORCH_AUDIO_VERSION=${{ inputs.torchaudio-version }}
-        ${{ inputs.cuda-arch-support && format('BUILD_TORCH_CUDA_ARCH_LIST={0}', inputs.cuda-arch-support) || '' }}
+        BUILD_TORCH_CUDA_ARCH_LIST=7.0 7.5 8.0 8.6 8.9 9.0+PTX
```

Comment:
Was this intentional to move this out of being parameterized?

Reply:
Yes, because we hit the limit of 10 inputs per reusable workflow. In retrospect I think it would be better to pass more of these as a block, like `inputs.additional-build-args`, when they don't need special processing (like the ones that need to be separated out to be included in the image tag).

This one should also have its default moved to the Dockerfile, so that they don't get out of sync.

Reply:
Reworked in 68fbfd11691200e55d920709b9a4802b639b0c29.